lib/modules/manager/cargo/artifacts.spec.ts

```diff
@@ -132,6 +132,7 @@ describe('modules/manager/cargo/artifacts', () => {
         packageName: 'dep1',
         lockedVersion: '1.0.0',
         newVersion: '1.0.1',
+        updateType: 'lockfileUpdate' as UpdateType,
```

Comment:
```suggestion
        updateType: 'lockfileUpdate' satisfies UpdateType,
```
use satisfies

Reply:
I ended up needing to use both `as` and `satisfies` to satisfy the type checker.